inc/commonitiltask.class.php

```diff
@@ -1498,7 +1498,8 @@ static function showCentralList($start, $status = 'todo', $showgrouptickets = tr
                } else if ($itemtype == "ProblemTask") {
                   $title = __("Problem tasks to do");
                }
-               echo "<a href=\"".$CFG_GLPI["root_doc"]."/front/ticket.php?".
+               $itemtype_url = strtolower(str_replace("Task","",$itemtype));
+               echo "<a href=\"".$CFG_GLPI["root_doc"]."/front/".$itemtype_url.".php?".
```

Comment:
Indeed, the URL is not good. Maybe better to do something like:
```suggestion
               $linked_itemtype = str_replace("Task","",$itemtype);
               echo "<a href=\"".$linked_itemtype::getSearchURL()."?".
```

Reply:
Thanks, i'm not familiar with all glpi classes methods.